src/elements/content-sidebar/SidebarNav.js

```diff
@@ -46,52 +47,64 @@ const SidebarNav = ({
     isOpen,
     onNavigate,
 }: Props) => (
-    <div aria-label={intl.formatMessage(messages.sidebarNavLabel)} role="tablist">
-        {hasActivity && (
-            <SidebarNavButton
-                interactionTarget={SIDEBAR_NAV_TARGETS.ACTIVITY}
+    <div className="bcs-SidebarNav" aria-label={intl.formatMessage(messages.sidebarNavLabel)} role="tablist">
+        <div className="bcs-SidebarNav-tabs">
+            {hasActivity && (
+                <SidebarNavButton
+                    interactionTarget={SIDEBAR_NAV_TARGETS.ACTIVITY}
+                    isOpen={isOpen}
+                    sidebarView={SIDEBAR_VIEW_ACTIVITY}
+                    onNavigate={onNavigate}
+                    tooltip={<FormattedMessage {...messages.sidebarActivityTitle} />}
+                >
+                    <IconChatRound />
+                </SidebarNavButton>
+            )}
+            {hasDetails && (
+                <SidebarNavButton
+                    interactionTarget={SIDEBAR_NAV_TARGETS.DETAILS}
+                    isOpen={isOpen}
+                    sidebarView={SIDEBAR_VIEW_DETAILS}
+                    onNavigate={onNavigate}
+                    tooltip={<FormattedMessage {...messages.sidebarDetailsTitle} />}
+                >
+                    <IconDocInfo />
+                </SidebarNavButton>
+            )}
+            {hasSkills && (
+                <SidebarNavButton
+                    interactionTarget={SIDEBAR_NAV_TARGETS.SKILLS}
+                    isOpen={isOpen}
+                    sidebarView={SIDEBAR_VIEW_SKILLS}
+                    onNavigate={onNavigate}
+                    tooltip={<FormattedMessage {...messages.sidebarSkillsTitle} />}
+                >
+                    <IconMagicWand />
+                </SidebarNavButton>
+            )}
+            {hasMetadata && (
+                <SidebarNavButton
+                    interactionTarget={SIDEBAR_NAV_TARGETS.METADATA}
+                    isOpen={isOpen}
+                    sidebarView={SIDEBAR_VIEW_METADATA}
+                    onNavigate={onNavigate}
+                    tooltip={<FormattedMessage {...messages.sidebarMetadataTitle} />}
+                >
+                    <IconMetadataThick />
+                </SidebarNavButton>
+            )}
+            {hasAdditionalTabs && <AdditionalTabs key={fileId} tabs={additionalTabs} />}
+        </div>
+        <div className="bcs-SidebarNav-toggle">
```

Comment:
Consider something like `bcs-SidebarNav-footer`, in case we want to add more to it?